### ISSUE TITLE
ManyToMany fields are not working

### ISSUE THREAD

**Issue by kamilkijak** [eamigo86/graphene-django-extras#96]
Querying `ManyToMany` fields is not working for `DjangoListObjectField` and other ListFields. Example below (it's based on `testproject` from the repository, [you can see the commit here](https://github.com/kamilkijak/graphene-django-extras/commit/01ef25d32d7e01921e40db8ebb63e01be93a0165)):

* **testproject/schema.py**
```python
from django.contrib.auth.models import Group

class GroupType(DjangoObjectType):
    class Meta:
        model = Group
        description = " Type definition for a single group "
        filter_fields = {}

# the rest unmodified
```
* **GraphQL query**
```graphql
{
  allUsers {
    results {
      id
      lastName
      groups {
        name
      }
    }
  }
}
```

* **result**
```graphql
{
  "errors": [
    {
      "message": "Cannot query field \"name\" on type \"UserType\". Did you mean \"username\"?",
      "locations": [
        {
          "line": 7,
          "column": 9
        }
      ]
    }
  ]
}
```

@eamigo86 Am I missing something?

**Comment by obisimon** [comment]
Hi @eamigo86
with Django 2.1.4 it seems to work when get_related_model() is not called from get_model_fields() in graphene_django_extras/utils.py
Regards Simon

**Comment by eamigo86** [comment]
Hi @obisimon, thanks for the help, I appreciate it!!!

**Comment by obisimon** [comment]
Hi @eamigo86, sorry a copy/paste error of mine, it's the **get_reverse_fields()** called in get_model_fields() on line 80.

**Comment by eamigo86** [comment]
Hello, @obisimon and @kamilkijak, the issue is already solved in the latest version 0.4.3.

**Comment by kamilkijak** [comment]
@eamigo86 great! Thank you.

We could also consider adding a test for that. I believe adding something like GroupType from [the mentioned commit](https://github.com/kamilkijak/graphene-django-extras/commit/01ef25d32d7e01921e40db8ebb63e01be93a0165) and querying that field (even if it's empty) would be good enough for now.
